src/main/java/io/lakefs/routerfs/RouterFileSystem.java

```diff
@@ -14,23 +16,29 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+@NoArgsConstructor
 public class RouterFileSystem extends FileSystem {
 
-    public static final Logger LOG = LoggerFactory.getLogger(RouterFileSystem.class);
+    private static final Logger LOG = LoggerFactory.getLogger(RouterFileSystem.class);
     private static final String DEFAULT_FS_CONF_PATTEREN = "^routerfs\\.default\\.fs\\.(?<fromScheme>[-a-z0-9_]*)";
```

Comment:
Not part of this PR, but
```suggestion
    private static final String DEFAULT_FS_CONF_PATTERN = "^routerfs\\.default\\.fs\\.(?<fromScheme>[-a-z0-9_]*)";
```
❓